callouts/java/service-callout/README.md

```diff
@@ -185,7 +185,8 @@ public static void main(String[] args) throws Exception {
     // Create a builder for ServiceCallout with custom configuration
     Example server = new Example.Builder()
             .setIp("111.222.333.444")       // Customize IP
-            .setPort(8443)                  // Set the port for secure communication
+            .setEnableTls(true)             // Enable TLS
+            .setPort(443)                   // Set the port for secure communication
```

Comment:
If you end up renaming `port` to `securePort` (per other comment), it would make sense to also rename the builder method to `setSecurePort`.